internal/tunnel/onboard.go

```diff
@@ -47,8 +54,8 @@ func Onboard(
 	}
 	defer client.Close()
 
-	// -- Make the data dir --
-	err = runCommand(client, "mkdir -p "+conf.DataDir)
+	// -- Make the install dir --
+	err = runCommand(client, "mkdir -p "+conf.installDir())
```

Comment:
Could we rename the `DataDir` field to `DataDirTemplate`, so it's clear that it shouldn't be used on it's own?